lib/OpenQA/Task/Asset/Limit.pm

```diff
@@ -89,8 +90,15 @@ sub _limit {
             my $age_in_seconds = ($now->epoch() - DateTime::Format::Pg->parse_datetime($asset->{t_created})->epoch());
             my $asset_name     = $asset->{name};
             if ($age_in_seconds >= $untracked_assets_storage_duration) {
+                my $storage_duration = $untracked_assets_storage_duration;
+                for my $pattern ($untracked_assets_patterns) {
+                    if ($asset_name =~ $pattern) {
+                        $storage_duration = $untracked_assets_patterns->{$pattern} * $seconds_per_day;
```

Comment:
I don't understand why you do this calculation just to revert it again two lines later

Reply:
Yeah. This is just from how the code was done before. The draft state on this PR really means, I wanted a basis for conversation before cleaning it up not knowing if we wanna go forward with this approach.